llvm/lib/Transforms/Utils/InjectTLIMappings.cpp

```diff
@@ -109,11 +109,11 @@ static void addMappingsFromTLI(const TargetLibraryInfo &TLI, CallInst &CI) {
   TLI.getWidestVF(ScalarName, WidestFixedVF, WidestScalableVF);
 
   for (bool Predicated : {false, true}) {
-    for (ElementCount VF = ElementCount::getFixed(2);
```

Comment:
Leave a comment about this change. It is unclear why this change is needed and whether it affects other vector libraries.